components/developer/rust/Makefile

```diff
@@ -73,12 +76,16 @@ CXXFLAGS +=	-D_POSIX_PTHREAD_SEMANTICS
 PKG_MACROS+= RUST_ARCH="$(RUST_ARCH)"
 
 COMPONENT_PREP_ACTION = \
-	$(GSED) -i -e 's/9c6af198a2f80da827054784faf70bc404673d9568fb45664b4f73c46245fdbd/c3851969a854d6ccaa8a94369f1288025149d1ebdc74b36f224423f35b18f7a7/' \
+	$(GSED) -i -e 's/1e732c2e3b4bd1561f11e0979bf9d20669a96eae7afe0deff9dfbb980ee47bf1/55abd8100db14a076dedbf84ce0f2db08158e1bd33ff1d4978bd3c4ad978f281/' \
+		$(SOURCE_DIR)/vendor/rand-0.6.1/.cargo-checksum.json; \
+	$(GSED) -i -e 's/8e5369ed1e86c5be92831d290b9eeb77c537a2de728950a5086e77438a4f0f3a/dac34dab563a2ca03c1d43f0224736f30662c3bbf0c2f1c44e220a7172345f4f/' \
 		$(SOURCE_DIR)/vendor/libc/.cargo-checksum.json; \
-	$(GSED) -i -e 's/cb94a0b54e47022b2e9b58b45e0196785c61d083760333276226da435cc040d9/bcb195562cf830c9e2078757e6d0976f86ede59816ff305e21a2764a4a4cf43e/' \
-		$(SOURCE_DIR)/vendor/rand/.cargo-checksum.json;
+	$(GSED) -i -e 's/59763fc255248b54fba5d0761d61093a73d51fa4cb400b0df1b5f339b9c2f48a/ba66d192421fd536ceddb50616c4c4aea06f4e39450eb0bc2bbbaed0b1e684c1/' \
+		$(SOURCE_DIR)/vendor/backtrace-sys/.cargo-checksum.json; \
+	$(GSED) -i -e 's/51a9ca7a4576822dc4fa01b31bc4bbed21817777abb304f605ef5951f68c3ed2/807c3aa4b5a62a70ba790df151e9ecf19f4bb562f2858575d690908751262db3/' \
+		$(SOURCE_DIR)/vendor/lzma-sys/.cargo-checksum.json;
 
-COMPONENT_PRE_CONFIGURE_ACTION = ($(CLONEY) $(SOURCE_DIR) $(@D))
+COMPONENT_PRE_CONFIGURE_ACTION = (rmdir $(@D); cp -a $(SOURCE_DIR) $(@D))
```

Comment:
Why do we nee rmdir $(@D) and not just cp -a?

Reply:
Well my scientific degree of cp -a was only sufficient enough to copy the source directory into @D as a subdirectory. this was the only solution I could come up with the would copy it in the right place and copy any dotfiles aswell. The other solution would be $(SOURCE_DIR)/* but that can sometimes break cp if it is too many files or if there are dotfiles.

Reply:
I see a lot of 
```
COMPONENT_PRE_CONFIGURE_ACTION =        (cp -a $(SOURCE_DIR)/* $(@D))
```
in oi-userland. If it's not a problem in this case, let's stick to it.

Reply:
Sure, done